src/app/action/action.html

```diff
@@ -18,4 +18,3 @@
     </form>
```

Comment:
please do not delete the last blank line. always keep a newline character at end of each file. 
* http://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file/18789
* http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline